tests/core/test_tracker_stores.py

```diff
@@ -559,12 +556,12 @@ def test_tracker_store_retrieve_with_session_started_events(
 ):
     tracker_store = tracker_store_type(default_domain, **tracker_store_kwargs)
     events = [
-        UserUttered("Hola", {"name": "greet"}),
-        BotUttered("Hi"),
-        SessionStarted(),
-        UserUttered("Ciao", {"name": "greet"}),
+        UserUttered("Hola", {"name": "greet"}, timestamp=1),
```

Comment:
do we need this because the timestamps on Windows would be the same?

Reply:
yes, all the timestamps on windows are the same, and the order is not consistent